bentoml/cli/deployment.py

```diff
@@ -233,6 +233,21 @@ def deployment():
     @click.option(
         '--replicas',
         help='Number of replicas. Option applicable to platform: Kubernetes',
+        type=click.INT,
+    )
+    @click.option(
+        '--memory-size',
+        help='Memory size for lambda function. '
+        'Option applicable to platform: aws-lambda',
+        type=click.INT,
+        default=1024,
+    )
+    @click.option(
+        '--timeout',
+        help='function timeout for lambda function. '
```

Comment:
nit: ""The maximum execution time (timeout) for AWS Lambda function. Option applicable to platform: aws-lambda.""

Reply:
what's the default value? is the int value in seconds or minutes? what are the limits? I believe it's 15minutues? Could you add that information here and ideally also add a related validator?